_disability-benefits/filing/active-duty/bdd.md

```diff
@@ -39,7 +39,7 @@ You must give your service treatment records to VA when you apply, and you must
 
 BDD is available nationwide and open to all Servicemembers on full-time active duty, including members of the National Guard and Reserve. Members of the Coast Guard may also participate.
 
-**Exception:** If you are less than 60 days from separation, you can submit a [Quick Start claim](/disability-benefits/filing/active-duty/quick-start/).
+**Exception:** If you are less than 60 days from separation, you can submit a [Quick Start claim](/disability-benefits/active-duty/quick-start/).
```

Comment:
I am not able to load this URL locally - thought I am if I add the old /filing/ again